compose-oneliner.sh

```diff
@@ -12,9 +12,9 @@ HOME_DIR=`eval echo ~$(logname)`
 DOCKER_COMPOSE_DIR=${HOME_DIR}/docker-compose
 COMPOSE_BASH_URL="https://github.com/AnyVisionltd"
 BRANCH=$1
-PRODUCT=$2
```

Comment:
please work on commit message .. 
how does changing environment variable helps ? can you please describe the problem and how does the solution solves it ?

Reply:
I have optional commits sso they need to come last.